CONTRIBUTING.md

```diff
@@ -11,26 +28,118 @@
+and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `main`. To contribute,
+[fork](https://help.github.com/articles/fork-a-repo/) this repository, commit your
+changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
 
+Note that our [code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md)
+applies to all platforms and venues related to this project; please follow it in all
+your interactions with the project and its participants.
 
 ## Observability Pack Guidelines
 
-We encourage all contributors to actively engage in the creation and maintenance of Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+We encourage all contributors to actively engage in the creation and maintenance of
+Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+
+- `Step 1`: Review the [pack template config](./_template/config.yml) file for a definition of how to create a pack.
+- `Step 2`: Review the [Pack Config File Schema](https://github.com/newrelic/newrelic-observability-packs/blob/f3acfd3efc34899c088c01b48c2e2f2210dd2657/docs/config-spec.md) for structure and limits you need to consider.
+- `Step 3`: Create your pack!
+- `Step 4`: Submit a PR.
+- `Step 5`: resolve feedback from code reviews
+- `Step 6`: after approval, merge your PR
+
+When creating a new Observability Pack or reviewing a PR please keep the following in mind, and refer to the
+[Packs validation workflow](https://github.com/newrelic/newrelic-observability-packs/blob/main/.github/workflows/validate_packs.yml) for up to date validations.
+
+### Required field best practices
+
+#### Name
+
+> The official name of the pack.
+
+- Required
+- limited to 100 characters
+- The `Validate Packs / Ensure pack names are unique` job verifies name uniqueness.
+- Avoid PR collisions by checking if any [open PRs](https://github.com/newrelic/newrelic-observability-packs/pulls) are using the same name you wish to use.
+- Pack names are stripped of any punctuation and white space is replaced by `-` before doing any comparisons.
+
+#### Description
+
+> A detailed description of the pack and why it's useful.
+
+- Required
+- Plain text only
+- limited to 2000 characters
+- The `TBD` job verifies pack descriptions
+
+#### Level
+
+> The support level provided for the pack.
+
+- Required
+- set to `Community` level by default
+- Can only be modified by New Relic employees to set another level type
+- The `TBD` job verifies pack level
+
+#### Author
+
+> The creator(s) of the pack
+
+- Required
+- No limit to the amount of authors
+- The `TBD` job verifies pack level
+
+### Optional field best practices
+
+Other fields in a pack are optional but it's recommended you consider the following to offer the best possible pack
+experience for users.
+
+#### Short Description
```

Comment:
This is `summary` now (see [config-spec.md](https://github.com/newrelic/newrelic-observability-packs/blob/main/docs/config-spec.md))